### ISSUE TITLE
[I/O site] 500 page is rendering after a 404 page is hit

### ISSUE THREAD

**Issue by jpvajda** [newrelic/instant-observability-website#97]
### Steps to reproduce

go to any 404 url, like https://instantobservabilitywebsitefea.gatsbyjs.io/40404040  and the 404 page renders but then the 500 page renders shortly after

### Expected 

the 500 page does not render when a user hit's a 404.

### Screen shot 

<img width="1420" alt="Screen Shot 2022-03-14 at 11 38 14 AM" src="https://user-images.githubusercontent.com/1395158/158233189-a7e7bba8-e07e-4d9c-a8bc-440ee43ceba8.png">

**Comment by jpvajda** [comment]
We decided to not launch the SSR feature at this time.

REASONS: 

- The performance issues we are seeing with category and page data and search results is too severe for a marketing site. Load times are anywhere from 1 - 5 seconds before caching  and spending time to optimize that doesn’t seem worth it at the moment 
- We are seeing large UI deviations in the internal and external catalogs UI  which is making this service less of a value add to the external site now
- Due to our change in direction internally, our original goal of having parity isn’t valid anymore
- As we learned from our GA releases: Our approach to the site has changed a lot in the past several months, to be strictly marketing focused and not something that needs to be a direct parity of the internal catalog, thus reducing the complexity. 
- This site is transitioning to a team focused on marketing website development, and SSR and an integration with the catalog service is extra complexity that offers little end user value, for just a marketing site
